### ISSUE TITLE
Double-click to move items in Local Filter panel

### ISSUE THREAD

**Issue by KevinThiele** [AtlasOfLivingAustralia/ala-keys-ui#16]
Would be handy to be able to move individual items between panels in the local filter popup by double-clicking as well as select-and-move-button. And - any chance of drag and drop for this?

**Comment by nielsklazenga** [comment]
Yes, absolutely. We'll have to do something like that anyway, if we want it to work on mobile devices, as they don't have the multiple selects (they are turned into groups of checkboxes). I like the multiple selects that are currently used because of the option to select multiple items at once using the CTRL and SHIFT keys, so I will have to see if I can replicate that in DIVs or ULs.

Drag-and-drop requires JQuery UI, which is currently not used in the KeyBase plugin. I am trying to keep the dependencies of the KeyBase plugin to a minimum, so drag-and-drop will probably not be part of the default functionality of the plugin. It will be in KeyBase itself though, as I like the idea.

Not going to happen right-away.
